tests/fixture/tmpnet/network.go

```diff
@@ -60,8 +60,11 @@ const (
 	// eth address: 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
 	HardHatKeyStr = "56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
 
+	// Default base grafana URI.
+	DefaultBaseGrafanaURI = "https://grafana-poc.avax-dev.network/"
+
 	// Default grafana URI used to construct metrics links. Can be overridden by setting GRAFANA_URI env var.
-	defaultGrafanaURI = "https://grafana-poc.avax-dev.network/d/kBQpRdWnk/avalanche-main-dashboard"
+	defaultGrafanaURI = DefaultBaseGrafanaURI + "d/kBQpRdWnk/avalanche-main-dashboard"
```

Comment:
nit: can we use a more meaningful variable name here than `defaultGrafanaURI` ? This is pretty similar to `DefaultBaseGrafanaURI`, but something like `defaultGrafanaMainDashboardURI` would be more descriptive as the previous one makes clear it's the base, but this doesn't make clear how it extends the base.